src/components/HtmlInjection.tsx

```diff
@@ -38,7 +38,7 @@ function useScripts(html: Props['html'], onError: Props['onError']) {
             document.body.appendChild(script);
         });
 
-        iframely?.load();
+        (window as any).iframely?.load(); // call iframely on global window object
```

Comment:
This doesn't look pretty. Does it? :D 

How about more verbose, but cleaner notation?

```suggestion
        if (typeof iframely !== 'undefined') {
            iframely.load();
        }
```

This way it'll work well with the global type definition for Iframely: https://github.com/prezly/content-renderer-react-js/blob/7311a7d1fb84fecc7aee924fd9171070cbbdef2e/src/globals.d.ts#L1-L5

Reply:
@e1himself sure! making the change